.github/workflows/main.yml

```diff
@@ -0,0 +1,101 @@
+name: CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches:
+      - master
+
+  release:
+    types: [published]
+
+  pull_request:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build-and-test:
+    runs-on: ubuntu-20.04
+    name: Build and Test
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.19' # The Go version to download (if necessary) and use.
+      - name: Install Build Dependencies
+        run: make get-build-deps
+      - name: Download required modules
+        run: make download
+      - name: Vet
+        run: make vet
+      - name: Lint
+        run: make lint
+      - name: Test
+        run: make test
+        env:
+          SMTP_HOST: ${{ secrets.SMTP_HOST }}
+          SMTP_PORT: ${{ secrets.SMTP_PORT }}
+          SMTP_USER: ${{ secrets.SMTP_USER }}
+          SMTP_PASS: ${{ secrets.SMTP_PASS }}
+      - name: Build
+        run: |
+          set -ex
+          for tool in ssl-checker smtp-checker; do
+            make $tool-linux-amd64
+            make $tool-linux-arm64
+            make $tool-darwin-amd64
+            make $tool-windows-amd64.exe
```

Comment:
For consistency, could you use ${tool}- instead of $tool-?

Reply:
Right, changing